catroid/src/androidTest/java/org/catrobat/catroid/test/BricksHelpUrlTest.java

```diff
@@ -444,8 +446,12 @@
 				"https://wiki.catrobat.org/bin/view/Documentation/BrickDocumentation/PaintNewLookBrick");
 		brickToHelpUrlMapping.put("org.catrobat.catroid.content.bricks.PlayNoteForBeatsBrick",
 				"https://wiki.catrobat.org/bin/view/Documentation/BrickDocumentation/PlayNoteForBeatsBrick");
+		brickToHelpUrlMapping.put("org.catrobat.catroid.content.bricks.OpenUrlBrick",
+				"https://wiki.catrobat.org/bin/view/Documentation/BrickDocumentation/OpenUrlBrick");
 		brickToHelpUrlMapping.put("org.catrobat.catroid.content.bricks.CopyLookBrick",
 				"https://wiki.catrobat.org/bin/view/Documentation/BrickDocumentation/CopyLookBrick");
+		brickToHelpUrlMapping.put("org.catrobat.catroid.content.bricks.EditLookBrick",
+				"https://wiki.catrobat.org/bin/view/Documentation/BrickDocumentation/EditLookBrick");
 	}
 
 	@Parameterized.Parameters(name = "{0}")
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.